index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="first-navbar">
+            <a href="https://www.facebook.com"> <img src="./images/facebook_icon.png"> </a>
+            <a href="https://www.facebook.com"> <img src="./images/twitter_icon.png"> </a>
+        </div>
+        <nav>
+            <div class="logo"> <a href="index.html"></a>
+            </div>
+            <div class="nav-bar">
+                <span class="hamburger-icon1">&#9776</span>
+                <span> <a href="index.html"> <img src="./images/logo.jpg" alt=""  class="header-logo" srcset=""></a></span>
+                <span class="cross hidden">X</span>
+            <ul class="second-navbar">
+                <li class="second-navbar-links">About</li>
+                <li class="second-navbar-links"><a href="#"></a> Products</li>
+                <li class="second-navbar-links"><a href="#"></a> News</li>
+                <li class="second-navbar-links"><a href="#"></a> login</li>
+            </ul>
+        </div>
+            
+        <div>
+            <ul class="second-navbar-popup hidden">
+                <li class="second-navbar-links"><a href="about.html">About</a> </li>
+                <li class="second-navbar-links"><a href="#"></a> Products</li>
+                <li class="second-navbar-links"><a href="#"></a> News</li>
+                <li class="second-navbar-links"><a href="#"></a> login</li>
+            </ul>
+        </div>
+        </nav>
+        <div class="banner-container">
+        <div class="heading">
+            <h4 class="heading-1">Hello!!!</h4>
+            <h1 class="heading-2"> Welcome To Online Makeup Blog </h1>
+        </div>
+        <div>
+            <p  class="paragraph">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Explicabo, libero?
+            </p>
+            <h4 class="heading-3">2023.10.15(THU) ~ 16(FRI)</h4>
+            <p class="heading-4">@ National Museum of Korea, Art center Nabi and more<p>
+        </div>
+    
+        </div>
+    </header>
+    <div class="main">
+<div class="main-heading flex">
+    <h2 class="main-head-text">
+        Evoltuion Of Makeup
```

Comment:
- [ ] For the Home page on the Desktop Size, the following is expected: 

- The middle section article list is organized in a horizontal row.
- The middle section individual articles have their elements organized in a vertical column, with the image at the top, the title in the center, and the description at the bottom.
- The bottom section articles are organized in two columns of three articles each.

Kindly implement this.